CMakeLists.txt

```diff
@@ -292,15 +292,18 @@ if(ENABLE_IPV6 AND NOT WIN32)
         CACHE BOOL "Define if you want to enable IPv6 support" FORCE)
   endif()
 
-  if(CMAKE_SYSTEM_NAME STREQUAL "Darwin" AND NOT ENABLE_ARES)
-    set(use_core_foundation_and_core_services ON)
+  if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
+    list(APPEND CURL_LIBS "-licucore")
```

Comment:
In context of `icucore` shouldn't it use `if(APPLE)` instead?

Reply:
You are right. It doesn't need to be in the block for "ENABLE_IPV6 AND NOT WIN32" and Darwin and not ARES.